### DISCUSSION TITLE
How do I use IronPython for this?

### DISCUSSION THREAD

**Discussion by DedFishy** [CosmosOS/Cosmos#2003] [Q&A]
How do I program this using IronPython?

**Comment by Doonort3** [comment]
Look up all the information on the Internet about it, and only then ask the 7-word question.

https://ironpython.net/documentation/
https://www.red-gate.com/simple-talk/development/dotnet-development/ironpython-first-steps/

**Reply by DedFishy** [reply]
I mean, how do I use IronPython to program using Cosmos?

**Comment by zarlo** [comment]
right now you cant there is no eta on when this would be doable

**Reply by DedFishy** [reply]
Ok, thanks!

**Reply by quajak** [reply]
I dont think its that difficult simply reference the nuget packages, define a kernel class and then build the project.
